central/risk/manager/manager.go

```diff
@@ -263,6 +265,14 @@ func (e *managerImpl) reprocessNodeComponentRisk(nodeComponent *storage.Embedded
 		return
 	}
 
+	oldScore := e.nodeComponentRanker.GetScoreForID(
+		scancomponent.ComponentID(nodeComponent.GetName(), nodeComponent.GetVersion(), os))
+
+	// Node risk results are not currently used so if the score is the same then no need to upsert
+	if oldScore == risk.GetScore() {
+		return
+	}
```

Comment:
Flip the order.

Reply:
Not sure I'm following on the order flip?

Reply:
Ah, `GetScoreForID` is not a datastore op. I misread it.